app/src/pages/lloydGeorgeUploadPage/LloydGeorgeUploadPage.test.tsx

```diff
@@ -149,6 +169,8 @@ describe('UploadDocumentsPage', () => {
             mockS3Upload.mockReturnValue(Promise.resolve());
             mockVirusScan.mockReturnValue(DOCUMENT_UPLOAD_STATE.CLEAN);
             mockUploadConfirmation.mockReturnValue(DOCUMENT_UPLOAD_STATE.SUCCEEDED);
+            mockUpdateDocumentState(Promise.resolve());
```

Comment:
```suggestion
```
I think we shouldn't have this line, because this is not mocking updateDocumentState, but calling the mocked function directly in the test.
When I removed this line locally, the test still passed, so probably it isn't needed as well.